docs/connector-development/connector-builder-ui/tutorial.mdx

```diff
@@ -29,11 +29,10 @@ The output schema of our stream will look like the following:
 ### Setting up Exchange Rates API key
 
 Before we get started, you'll need to generate an API access key for the Exchange Rates API.
-This can be done by signing up for the Free tier plan on [Exchange Rates API](https://exchangeratesapi.io/):
+This can be done by signing up for the Free tier plan on [Exchange Rates API](https://apilayer.com/marketplace/exchangerates_data-api):
 
-1. Visit https://exchangeratesapi.io and click "Get free API key" on the top right
-2. You'll be taken to https://apilayer.com -- finish the sign up process, signing up for the free tier
-3. Once you're signed in, visit https://apilayer.com/marketplace/exchangerates_data-api#documentation-tab and click "Live Demo"
+1. Visit https://apilayer.com and click Sign In on the top right - either sign into an existing account or sign up for a new account on the free tier.
+3. Once you're signed in, visit https://apilayer.com/marketplace/exchangerates_data-api and click "Live Demo".
```

Comment:
No need to click "Live Demo" anymore and no need for navigating its editor in the following step either: once you're signed in, https://apilayer.com/marketplace/exchangerates_data-api shows your API key with a "copyAPI key" CTA right beside now. It does still work to go into the live demo editor and copy the string from there, but, like, why?
<img width="1090" alt="Screenshot 2023-10-30 at 3 16 43 PM" src="https://github.com/airbytehq/airbyte/assets/7516653/83f75629-99e4-4d9b-9a23-42a1b8c24b01">

Reply:
Good catch, fixed!